src/test/java/cosminsechel/tests/seleniumpart1/Demoqatema.java

```diff
@@ -0,0 +1,29 @@
+package cosminsechel.tests.seleniumpart1;
+
+import cosminsechel.driver.BrowserManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class Demoqatema {
+
+    public static void main(String[] args) throws InterruptedException {
+        BrowserManager.createDriver();
+        WebDriver driver = new ChromeDriver();
+        driver.get("https://demoqa.com/elements");
+        driver.manage().window().maximize();
+        driver.findElement(By.id("item-0")).click();
+        driver.findElement(By.id("userName")).sendKeys("Cosmin");
+        driver.findElement(By.id("item-1")).click();
+        WebElement checkbox = driver.findElement(By.cssSelector("label[for='tree-node-home']"));
+        checkbox.click();
+        if (checkbox.isSelected()) {
+            System.out.println("checkbox selected");
+
+        }else{
+            System.out.println("checkbox not selected ");
+        }
+
```

Comment:
Please close the browser at the end. You can use the method defined in BrowserManager.